app/src/main/java/com/kylecorry/trail_sense/navigation/ui/NavigatorFragment.kt

```diff
@@ -631,9 +631,9 @@ class NavigatorFragment : BoundFragment<ActivityNavigatorBinding>() {
 
     private fun updateCompassView() {
         val destBearing = getDestinationBearing()
-        val destColor = destination?.color ?: Resources.color(
+        val destColor = destination?.color ?: Resources.getAndroidColorAttr(
```

Comment:
This should remain R.color.colorAccent since the compass background doesn't change color based on the theme (to ensure visibility)

Reply:
Done